.github/workflows/main-cmake.yml

```diff
@@ -0,0 +1,251 @@
+name: hdf5 dev CMake CI
```

Comment:
name should be hdf5 1.14 in all these yaml files

Reply:
will do